diff -up dump-0.4b42/dump/tape.c.rh702593 dump-0.4b42/dump/tape.c
--- dump-0.4b42/dump/tape.c.rh702593	2011-05-17 12:39:23.414859785 +0200
+++ dump-0.4b42/dump/tape.c	2011-05-17 12:43:56.102387189 +0200
@@ -211,8 +211,17 @@ static int gtperr = 0;
  * Neither is useful in our case, so this is easy to handle.
  */
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,49)
-/* clone_flags, child_stack, parent_tidptr, child_tidptr */
+/* 
+ * Parameters of the sys_clone syscall are
+ * clone_flags, child_stack, parent_tidptr, child_tidptr
+ * on all architectures except s390 and s390x
+ * s390* have child_stack, clone_flags, parent_tidptr, child_tidptr
+ */
+#if defined(__s390__) || defined(__s390x__)
+#define CLONE_ARGS 0, SIGCHLD|CLONE_IO, NULL, NULL
+#else
 #define CLONE_ARGS SIGCHLD|CLONE_IO, 0, NULL, NULL
+#endif
 #else
 #define CLONE_ARGS SIGCHLD|CLONE_IO, 0
 #endif /* LINUX_VERSION_CODE */
