--- dump-0.4b42/dump/optr.c.ext4	2006-03-14 12:09:51.000000000 +0100
+++ dump-0.4b42/dump/optr.c	2010-12-02 15:52:09.752382850 +0100
@@ -78,7 +78,7 @@ int	datesort __P((const void *, const vo
 static	void sendmes __P((const char *, const char *));
 
 /* List of filesystem types that we can dump (same ext2 on-disk format) */
-static char *fstypes[] = { "ext2", "ext3", "InterMezzo", NULL };
+static char *fstypes[] = { "ext2", "ext3", "ext4", "InterMezzo", NULL };
 
 /*
  *	Query the operator; This previously-fascist piece of code
