Index: dump/traverse.c
===================================================================
RCS file: /cvsroot/dump/dump/dump/traverse.c,v
retrieving revision 1.71
retrieving revision 1.72
diff -u -r1.71 -r1.72
--- dump/traverse.c	10 Jun 2010 12:17:35 -0000	1.71
+++ dump/traverse.c	21 Feb 2011 10:36:47 -0000	1.72
@@ -992,7 +992,7 @@
 	bc.buf = (int *)malloc (bc.max * sizeof (int));
 	bc.cnt = 0;
 	bc.ino = ino;
-	bc.next_block = NDADDR;
+	bc.next_block = 0;
 
 	ext2fs_block_iterate2(fs, (ext2_ino_t)ino, BLOCK_FLAG_DATA_ONLY, NULL, dumponeblock, (void *)&bc);
 	/* deal with holes at the end of the inode */
